parsing/lexer.mll

```diff
@@ -377,7 +378,11 @@ rule token = parse
         COMMENT (s, loc) }
   | "(**"
       { let s, loc = with_comment_buffer comment lexbuf in
-        DOCSTRING (Docstrings.docstring s loc) }
+        if !handle_docstrings then
+          DOCSTRING (Docstrings.docstring s loc)
+        else
+          COMMENT (s, loc)
```

Comment:
For the sake of coherence is indeed probably better to keep the leading "*". 
It does not affect ocamldoc, since ocamldoc reimplements all the comment parsing logic on its side. Droping the comment feels like a quite radical solution compared to just deactivating the special handling of docstring.